steve/util.py

```diff
@@ -54,6 +93,39 @@ def load(self):
     def get(self, section, key):
         raise NotImplementedError
 
+    def create(self, dir_path):
+        """create the config file for the project"""
+        abs_path = os.path.abspath(dir_path)
+        kw = dict(
+            projectpath=abs_path,
+            jsonpath=os.path.join(abs_path, 'json'),
+            yamlpath=os.path.join(abs_path, 'yaml'),
+            status=self.default_status,
+        )
+        with open(os.path.join(dir_path, self.file_name), 'w') as fp:
+            fp.write(self._CONFIG.format(**kw))
+
+    @property
+    def status(self):
+        """retrieve status information"""
+        sfn = self.status_file_name
+        try:
+            return yaml_load(open(sfn))
+        except IOError:
```

Comment:
Is this to handle the situation where the status file doesn't exist? If so, it's better to do:

```
if os.path.exists(sfn):
    return yaml_load(open(sfn))
return yaml_load(textwrap.dedent("""\
...
```

That way you're not accidentally stomping on the status file in situations where it exists, but there's some other problem.